.github/workflows/integration-system-tests.yml

```diff
@@ -197,8 +197,7 @@ jobs:
           use-uv: ${{ inputs.use-uv }}
       - name: "System Tests"
         run: >
-          ./scripts/ci/testing/run_system_tests.sh
-          tests/system/example_empty.py providers/tests/system/example_empty.py
+          ./scripts/ci/testing/run_system_tests.sh tests/system/example_empty.py
```

Comment:
We do not distinguish "core" and "providers" system tests any more. They are essentially the same.